narayana/ArjunaJTA/jta/tests/classes/org/jboss/narayana/rts/EmptyTxnTest.java

```diff
@@ -50,14 +48,14 @@ public static void tearDown() throws IOException {
 
     @Benchmark
     public void testEmptyTxn(Blackhole bh) throws IOException {
-        bh.consume(beginTx());
-        // make two service calls for comparison with RTSTests#testTxn
-        bh.consume(sendRequest());
-        bh.consume(TxnHelper.endTxn(txnClient, links));
+        bh.consume(testEmptyTxn());
     }
 
-    public boolean beginTx() throws IOException {
```

Comment:
Please keep in mind I don't need this 'beginTx' method anymore since the links variable is not global. In fact I need to assign that variable in a different way (line 55).